src/renderer/features/assethub-migration-modal/components/AssetHubMigrationModal.tsx

```
@@ -0,0 +1,105 @@
+import { useUnit } from 'effector-react';
+
+import { type ChainId } from '@/shared/core';
+import { useI18n } from '@/shared/i18n';
+import { RelayChains } from '@/shared/lib/utils/constants';
+import { Button } from '@/shared/ui/Buttons';
+import { InfoLink } from '@/shared/ui/InfoLink/InfoLink';
+import { FootnoteText, HeaderTitleText, SmallTitleText } from '@/shared/ui/Typography';
+import { ChainIcon } from '@/shared/ui-entities/ChainIcon/ChainIcon';
+import { Box, Modal } from '@/shared/ui-kit';
+import { networkModel } from '@/entities/network';
+import { migrationModalModel } from '../model/migrationModalModel';
+
+const getChainName = (chainId: ChainId): string => {
+  if (chainId === RelayChains.POLKADOT) return 'polkadot';
+  if (chainId === RelayChains.KUSAMA) return 'kusama';
+  return 'polkadot';
```

Is it correct to fallback to Polkadot for unknown chains?

---

Since everything is already hardcoded for these two modals, I think it’s fine. I just wanted to avoid adding extra null-handling logic elsewhere in the code

---

let's just throw an error, it's okay here